misk/web/modules/log4j/src/core/reducers/color.ts

```diff
@@ -0,0 +1,18 @@
+import { Action } from '../actions';
+import { COLOR, COLORS } from '../constants';
+
+const INITIAL_STATE = {
+  value: COLORS.DEFAULT
+};
+
+export default function color(state = INITIAL_STATE, action: Action) {
+  switch (action.type) {
+    case COLOR.CHANGE:
+      return Object.assign({}, {
```

Comment:
nit: shouldn’t this take state somewhere?

Reply:
(it’s academic ’cause it’s a single field, but I want to copy-paste this elsewhere and have it work)

Reply:
My understanding is that the color function does take state as a parameter and also includes a default `INITIAL_STATE` drawn on if state is not passed. Am I missing something? 

`color(state = INITIAL_STATE, ...`